### PR TITLE
fix: primefaces #6234, TreeSelect: (accessiblility) cannot access and clear the selected input using using keyboard

### PR DESCRIPTION
**Pull request by akshayaqburst**
… 

### Defect Fixes
fix: primefaces #6234, TreeSelect: (accessiblility) cannot access and clear the selected input using using keyboard

### CODE REVIEW

**Comment by akshayaqburst** [comment]
@melloware A PR for review, but its build is failing. **npm run lint:fix** in my local shows a lot of lint changes. Please advice.

**Comment by nitrogenous** [comment]
FYI This change planned for the next release. Thank you for your contribution

File: components/lib/treeselect/TreeSelect.js [hunk 1]
```diff
@@ -621,7 +628,10 @@ export const TreeSelect = React.memo(
                 const clearIconProps = mergeProps(
                     {
                         className: cx('clearIcon'),
-                        onPointerUp: clear
+                        onPointerUp: clear,
+                        tabIndex: props.tabIndex || '0',
+                        onKeyDown: onClearIconKeyDown,
+                        'aria-label': 'Clear Selection'
```
Comment (melloware) [inline components/lib/treeselect/TreeSelect.js:634]:
```suggestion
                        'aria-label': localeOption('clear')
```

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by akshayaqburst** [comment]
@melloware done

**Review by melloware — APPROVED** [review]

**Review by nitrogenous — APPROVED** [review]
LGTM

**Pull request merged**